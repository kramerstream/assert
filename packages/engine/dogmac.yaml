uses:
  assert:
    js: assert
  expected:
    js: @cosmokramer/expected
  path:
    js: @dogmalang/path
