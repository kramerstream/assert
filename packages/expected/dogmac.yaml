uses:
  assert:
    js: assert
  path:
    js: @dogmalang/path
